sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -325,7 +325,7 @@
       "post": {
         "summary": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language.",
         "description": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information. Pagination is supported. This returns information about device twins only.",
-        "operationId": "RegistryManager_QueryIotHub",
+        "operationId": "Devices_QueryIotHub",
```

Comment:
I don't see the query API as a part of our Devices API design PR; isn't Query going to have it's own client?

Reply:
Not as per the onenote description, did I miss something?

Reply:
I think we discussed it won't be its own client.